tests/bsim/bluetooth/audio/src/cap_initiator_broadcast_test.c

```diff
@@ -39,15 +39,15 @@
  * Broadcast ISO radio events.
  */
 #define BT_LE_EXT_ADV_CUSTOM                                                                       \
-	BT_LE_ADV_PARAM(BT_LE_ADV_OPT_EXT_ADV, BT_GAP_MS_TO_ADV_INTERVAL(80),                      \
-			BT_GAP_MS_TO_ADV_INTERVAL(80), NULL)
+	BT_LE_ADV_PARAM(BT_LE_ADV_OPT_EXT_ADV, BT_GAP_MS_TO_ADV_INTERVAL(140),                     \
+			BT_GAP_MS_TO_ADV_INTERVAL(140), NULL)
 
 #define BT_LE_PER_ADV_CUSTOM                                                                       \
-	BT_LE_PER_ADV_PARAM(BT_GAP_MS_TO_PER_ADV_INTERVAL(90), BT_GAP_MS_TO_PER_ADV_INTERVAL(90),  \
-			    BT_LE_PER_ADV_OPT_NONE)
+	BT_LE_PER_ADV_PARAM(BT_GAP_MS_TO_PER_ADV_INTERVAL(150),                                    \
+			    BT_GAP_MS_TO_PER_ADV_INTERVAL(150), BT_LE_PER_ADV_OPT_NONE)
 
 #define BROADCAST_STREMT_CNT    CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT
-#define BROADCAST_ENQUEUE_COUNT 2U
+#define BROADCAST_ENQUEUE_COUNT 18U
```

Comment:
This refers to the number of SDUs to enqueue - 18 seems excessive (180ms of audio data). Is that intentional?

Reply:
50 ms ISO interval with PTO=1 (BN=5 + 1) needs to have 12 SDUs buffered at an ISO interval and additional 6 SDUs to ensure subsequent ISO intervals have SDUs ready in the Controller. This is equivalent to how  3 Tx ACL buffers are required to ensure a peripheral ACL sends data every connection interval.